.github/workflows/ci.yml

```
@@ -0,0 +1,22 @@
+name: CI
+
+on: [push, pull_request]
+
+permissions:
+    contents: read
+
+jobs:
+    test:
+        name: Test on Node.js Latest
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v3
+            - name: Use Node.js latest
+              uses: actions/setup-node@v3
+              with:
+                node-version: latest
```

Let’s keep `.nvmrc` as the single source of truth, i.e. update the version number in it and then refer to it here:

```suggestion
      - name: Set up Node.js
        uses: actions/setup-node@v3
        with:
          node-version-file: '.nvmrc'
```

(also let’s use two-space indents?)